auth-foundation/src/main/java/com/okta/authfoundation/credential/Credential.kt

```diff
@@ -137,14 +215,29 @@ class Credential internal constructor(
 
     /**
      * The tags associated with this [Credential].
-     *
-     * This can be used when calling [Credential.storeToken] to associate this [Credential] with data relevant to your application.
      */
     val tags: Map<String, String>
         get() {
             return Collections.unmodifiableMap(state.value.tags)
         }
 
+    /**
+     * Set tags for this [Credential].
+     */
+    suspend fun setTags(tags: Map<String, String>) {
```

Comment:
Before, this functionality was provided by storeToken function. storeToken used to be replaceToken + setTags. I decided to separate them out.